lib/android/src/main/java/com/airbnb/android/react/maps/AirMapView.java

```diff
@@ -169,6 +173,24 @@ public boolean onScroll(MotionEvent e1, MotionEvent e2, float distanceX,
     });
 
     eventDispatcher = reactContext.getNativeModule(UIManagerModule.class).getEventDispatcher();
+
+    // Set up a parent view for triggering visibility in subviews that depend on it.
+    // Mainly ReactImageView depends on Fresco which depends on onVisibilityChanged() event
+    attacherGroup = new ViewAttacherGroup(context);
+    LayoutParams attacherLayoutParams = new LayoutParams(0, 0);
+    attacherLayoutParams.width = 0;
+    attacherLayoutParams.height = 0;
+    attacherLayoutParams.leftMargin = 99999999;
+    attacherLayoutParams.topMargin = 99999999;
+    attacherGroup.setLayoutParams(attacherLayoutParams);
+    attacherGroup.setVisibility(VISIBLE);
+    attacherGroup.setAlpha(0.0f);
+    attacherGroup.setRemoveClippedSubviews(false);
+    if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.JELLY_BEAN_MR2) {
+      attacherGroup.setClipBounds(new Rect(0, 0, 0, 0));
+    }
+    attacherGroup.setOverflow(ViewProps.HIDDEN);
```

Comment:
I'm seeing an error from this line on install, @alvelig @danielgindi:

```
 error: cannot find symbol
    attacherGroup.setOverflow(ViewProps.HIDDEN);
                                       ^
  symbol:   variable HIDDEN
  location: class ViewProps
Note: Some input files use or override a deprecated API.
Note: Recompile with -Xlint:deprecation for details.
Note: Some input files use unchecked or unsafe operations.
Note: Recompile with -Xlint:unchecked for details.
1 error


FAILURE: Build failed with an exception.
```

Reply:
@breadadams getting this error here too.

Reply:
Fixed in https://github.com/react-community/react-native-maps/commit/710dca49a515f0f781f51fcc6997af43da9923cc